Directory.Build.targets

```diff
@@ -19,10 +19,16 @@
   <Import Project="eng\NuGetVersions.targets" />
 
   <PropertyGroup>
-     <!-- Allows for MAUI Xaml Hot Reload Samples to run without checks  -->
+    <!-- Allows for MAUI Xaml Hot Reload Samples to run without checks  -->
     <IgnoreMauiXamlHotReloadCompatibilityCheck>True</IgnoreMauiXamlHotReloadCompatibilityCheck>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <!-- Microsoft.Build.Msix.props" cannot be imported again -->
+    <!-- Found version-specific or distribution-specific runtime identifier(s)-->
+    <NoWarn>$(NoWarn);MSB4011;NETSDK1206</NoWarn>
```

Comment:
this doesn't t actually seem to work, but would be good to try to hide these warnings or fix them @mattleibow